app.py

```diff
@@ -8,10 +9,12 @@
 app = Flask(__name__)
 
 
-BASE_URL = os.environ.get('BASE_URL',
-                          'https://cplab1.techservices.illinois.edu:443/api')
-CLIENT_ID = os.environ.get('CLIENT_ID')
-CLIENT_SECRET = os.environ.get('CLIENT_SECRET')
+BASE_URL: str = os.environ.get(
+    'BASE_URL',
+    'https://cplab1.techservices.illinois.edu:443/api'
+)
+CLIENT_ID: Optional[str] = os.environ.get('CLIENT_ID')
```

Comment:
I recommend you refactor this line as:

```python
CLIENT_ID: str = os.environ['CLIENT_ID']
```

Reply:
Thanks, I was struggling with this one. Optional did not seem right.